README.md

```diff
@@ -29,3 +29,21 @@ optional arguments:
                         Maximum duration (in minutes) to read data before
                         starting a new file
 ```
+
+
+## Development
+
+Project contains a buildable test to verify basic functionality. 
+Build and run with:
+
+```
+nix-build test.nix -A test
+./result/bin/nixos-test-driver
+```
+You should see a few messages from the service slurp: that it's running, that it found a
+topic, and then keeps searching.
+
+You can run it interactively with `nix-build slurp-test.nix -A test.driverInteractive`
+Here you can login as root (no password) and
+verify that /root/output contains a folder with a single compressed file.
```

Comment:
Make this the actual test instead of sleep, so we don't have to verify this manually.